### PR TITLE
Add info objects

### PR DESCRIPTION
**Pull request by anguera5**
· Add missing additional information objects
· Add missing docstrings

### CODE REVIEW

File: enviPath_python/objects.py [hunk 1]
```diff
@@ -2851,664 +3056,4265 @@ def parse(cls, data_string):
...
+    """
+    Creates a humidity additional information object.
+
+    This class represents additional information about experimental humidity.
+    """
+    name = "humidity"
+    mandatories = ["waterStorageCapacity"]
+
+    # Setter
+    def set_waterStorageCapacity(self, value):
```
Comment (anguera5) [inline enviPath_python/objects.py:4755]: Does this object have this parameter? Only one can be set from the UI

File: enviPath_python/objects.py [hunk 2]
```diff
@@ -2851,664 +3056,4265 @@ def parse(cls, data_string):
...
+
+        :param data_string: A string representing the location value.
+        :type data_string: str
+        :return: An instance of LocationAdditionalInformation populated with the parsed data.
+        :rtype: LocationAdditionalInformation
+        """
+        return cls._parse_default(data_string, ['location'])
+
+
+class ProposedIntermediateAdditionalInformation(AdditionalInformation):
```
Comment (anguera5) [inline enviPath_python/objects.py:5127]: How this exactly work on practice on the UI?

File: enviPath_python/objects.py [hunk 3]
```diff
@@ -2851,664 +3056,4265 @@ def parse(cls, data_string):
...
+        return cls(**res)
+
+
+class ReferringScenarioAdditionalInformation(AdditionalInformation):
+    name = "referringscenario"
+    mandatories = ['referringscenario']
+
+    # Setter
+    def set_referringscenario(self, value):
+        self.params["referringscenario"] = value
```
Comment (anguera5) [inline enviPath_python/objects.py:4248]: we should check that value is indeed a valid scenario

File: enviPath_python/objects.py [hunk 4]
```diff
@@ -2851,664 +3056,4265 @@ def parse(cls, data_string):
...
+
+    # Setter
+    def set_sourcescenario(self, value):
+        """
+        Sets the source scenario.
+
+        :param value: The source scenario as scenario ID.
+        :type value: str
+        """
+        self.params["sourcescenario"] = value
```
Comment (anguera5) [inline enviPath_python/objects.py:6450]: We should check that value is a valid scenario URL

File: enviPath_python/objects.py [hunk 5]
```diff
@@ -2851,664 +3056,4265 @@ def parse(cls, data_string):
...
+        """
+        Parses the data_string to create a SpikeCompoundAdditionalInformation instance.
+
+        :param data_string: A string containing spike compound data.
+        :type data_string: str
+        :return: SpikeCompoundAdditionalInformation instance.
+        :rtype: SpikeCompoundAdditionalInformation
+        """
+        res = {"spikeComp": data_string}
+        return cls(**res)
```
Comment (anguera5) [inline enviPath_python/objects.py:6517]: We cannot get the info of the spike compound if the only data stored on "value" is the compound name

File: enviPath_python/objects.py [hunk 6]
```diff
@@ -2620,216 +2623,418 @@ def parse(cls, data_string):
...
-
-    # Setter
-    def set_rateconstantlower(self, value):
-        self.params["rateconstantlower"] = value
-
-    def set_rateconstantupper(self, value):
-        self.params["rateconstantupper"] = value
+class AerationTypeAdditionalInformation(AdditionalInformation):
+    """
+    Creates an Aeration Type additional information object.
```
Comment (anguera5) [inline enviPath_python/objects.py:2922]: what's the difference of this with TypeOfAerationAdditionalInformation ?

**Review by lorsbach — APPROVED** [review]

**Pull request merged**